src/assets/icons/github.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg height="100mm" style="shape-rendering:geometricPrecision; text-rendering:geometricPrecision; image-rendering:optimizeQuality; fill-rule:evenodd; clip-rule:evenodd" viewBox="0 0 100 100" width="100mm" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><style type="text/css">
```

Comment:
this doesn't quite seem like the GitHub icon from the old side, which I think might look a bit better

Reply:
I wonder if we should just use `fontawesome` - see #46

Reply:
Let's use fontawesome